corehq/motech/repeaters/expression/forms.py

```diff
@@ -18,9 +24,34 @@ class CaseExpressionRepeaterForm(GenericRepeaterForm):
         help_text=_("Items to add to the end of the URL. Please see the documentation for more information.")
     )
 
+    case_action_filter_expression = JsonField(
+        expected_type=dict, required=False,
+        label=LABELS['case_action_filter_expression'],
+        help_text=_(
+            "Use this to determine if the response should create or update a case. "
+            "If left blank, the response will be ignored. "
```

Comment:
nit:
Should we reframe this to "if left blank, no action will be taken"?
I wonder if someone would read this and get confused if the request would not get registered/accepted.